model/src/cfd.rs

```diff
@@ -2982,7 +3058,9 @@ mod tests {
     }
 
     pub fn dummy_identity() -> Identity {
-        Identity::new(RandomSeed::default().derive_identity().0)
+        Identity::new(x25519_dalek::PublicKey::from(
+            *b"hello world, oh what a beautiful",
+        ))
```

Comment:
Stuff like this would be nice in a separate patch. Then the last commit could all just be "move files".